monkey/agent_plugins/exploiters/hadoop/src/hadoop_command_builder.py

```diff
@@ -78,11 +79,17 @@ def _build_linux_hadoop_command(
         download_url=agent_download_url,
     )
 
+    chmod_options = LinuxSetPermissionsOptions(
+        agent_destination_path=agent_destination_path, permissions=0o700
```

Comment:
I think it's better to not specify the default

Reply:
I was thinking that "Explicit is better than implicit".